core-ui/src/resources/Events/EventList.js

```diff
@@ -120,8 +120,14 @@ export function EventList(props) {
       hasDetailsView={true}
       readOnly={true}
       filter={res => {
-        if (displayType.key === EVENT_MESSAGE_TYPE.ALL.key) return true;
-        return res.type === displayType.key;
+        const typeFilter =
+          displayType.key === EVENT_MESSAGE_TYPE.ALL.key ||
+          res.type === displayType.key;
+
+        const propsFilter =
+          typeof props.filter === 'function' ? props.filter(res) : true;
```

Comment:
Just an idea, have you thought about extracting `filter` and `type` from the props? The `props` prefix won't be needed then.